### PR TITLE
Populate survey title into survey pages

### PR DESCRIPTION
**Pull request by weapdiv-david**
### Changes

Populate survey title into the beta banner, leaving the error pages blank
### How to test

1) Checkout the branch
2) View the survey title next to the beta button on each survey page
3) Cause an error and see the survey title is missing from next to the beta button
### Who can test

Anyone except @weapdiv-david and @warrenbailey

### CODE REVIEW

**Comment by LJBabbage** [comment]
A few code style check failures

File: app/main/views/questionnaire.py [hunk 1]
```diff
@@ -33,5 +33,6 @@ def questionnaire():
     return render_template('questionnaire.html', questionnaire=render_data, data={
         "survey_code": questionnaire_manager.get_schema().survey_id,
         "period_str": current_user.get_period_str(),
-        "respondent_id": current_user.get_ru_ref(),
-    })
+        "respondent_id": current_user.get_ru_ref()
+    },
+    bar_title="Monthly Business Survey - Retail Sales Index")
```
Comment (warrenbailey) [inline app/main/views/questionnaire.py]: bar_title=questionnaire_manager.get_schema().title

**Comment by warrenbailey** [comment]
This title is already in the schema so why not use that rather than hard coding it everywhere? 👎

File: app/main/views/thank_you.py [hunk 2]
```diff
@@ -33,4 +33,5 @@ def thank_you():
         "period_str": current_user.get_period_str(),
         "respondent_id": current_user.get_ru_ref(),
         "submitted_at": submitted_at
-    })
+    },
+    bar_title="Monthly Business Survey - Retail Sales Index")
```
Comment (warrenbailey) [inline app/main/views/thank_you.py]: bar_title=schema.title

File: app/main/views/submission.py [hunk 3]
```diff
@@ -56,5 +56,6 @@ def submission():
     return render_template('submission.html', questionnaire=render_data, data={
         "survey_code": schema.survey_id,
         "period_str": current_user.get_period_str(),
-        "respondent_id": current_user.get_ru_ref(),
-    })
+        "respondent_id": current_user.get_ru_ref()
+    },
+    bar_title="Monthly Business Survey - Retail Sales Index")
```
Comment (warrenbailey) [inline app/main/views/submission.py]: bar_title=schema.title

File: app/main/views/landing_page.py [hunk 4]
```diff
@@ -39,4 +39,5 @@ def landing_page():
         "return_by": '{dt.day} {dt:%B} {dt.year}'.format(dt=datetime.strptime(current_user.get_return_by(), "%Y-%m-%d")),
         "start_date": '{dt.day} {dt:%B} {dt.year}'.format(dt=datetime.strptime(current_user.get_ref_p_start_date(), "%Y-%m-%d")),
         "end_date": '{dt.day} {dt:%B} {dt.year}'.format(dt=datetime.strptime(current_user.get_ref_p_end_date(), "%Y-%m-%d"))
-    })
+    },
+    bar_title="Monthly Business Survey - Retail Sales Index")
```
Comment (warrenbailey) [inline app/main/views/landing_page.py]: bar_title=questionnaire.title

**Comment by warrenbailey** [comment]
Rebasing

File: tests/integration/mci/test_happy_path.py [hunk 5]
```diff
@@ -24,6 +24,7 @@ def happy_path(self,form_type_id):
         content = resp.get_data(True)
         self.assertRegexpMatches(content, '<title>Introduction</title>')
         self.assertRegexpMatches(content, '>Get Started<')
+        self.assertRegexpMatches(content, '<div class="bar__title">Monthly Business Survey - Retail Sales Index</div>')
```
Comment (hamishtaplin) [inline tests/integration/mci/test_happy_path.py]: Is it a good idea to include markup in our tests? If I change the markup it'll break the test in several places.

**Comment by hamishtaplin** [comment]
Can we remove markup from the tests before this is merged? I don't think the tests should be concerned with a particular class being used.

File: tests/integration/mci/test_happy_path.py (hunk 5 above)
Comment (warrenbailey) [inline tests/integration/mci/test_happy_path.py]: Agreed, i'll remove the class now

**Pull request merged**